packages/client/hmi-client/src/components/workflow/scenario-templates/situational-awareness/situational-awareness-template.ts

```diff
@@ -0,0 +1,82 @@
+import { BaseScenario } from '@/components/workflow/scenario-templates/scenario-template';
+import * as workflowService from '@/services/workflow';
+import { operation as ModelOp } from '@/components/workflow/ops/model/mod';
+import { operation as ModelConfigOp } from '@/components/workflow/ops/model-config/mod';
+import { operation as CalibrateCiemssOp } from '@/components/workflow/ops/calibrate-ciemss/mod';
+import { operation as DatasetOp } from '@/components/workflow/ops/dataset/mod';
+import { OperatorNodeSize } from '@/services/workflow';
+
+export class SituationalAwarenessScenario extends BaseScenario {
+	public static templateId = 'situational-awareness';
+
+	public static templateName = 'Situational Awareness';
+
+	modelSpec: { id: string };
+
+	datasetSpec: { id: string };
+
+	modelConfigSpec: { id: string };
+
+	calibrateSpec: { ids: string[] };
```

Comment:
Why all the spacing, this ain't Java 😝 
```suggestion
	public static templateId = 'situational-awareness';
	public static templateName = 'Situational Awareness';
	modelSpec: { id: string };
	datasetSpec: { id: string };
	modelConfigSpec: { id: string };
	calibrateSpec: { ids: string[] };
```

Reply:
The linter is doing it, I swear!